File: r/notes.R
controlTotals_message <- "(Standard errors are approximately zero for control totals)"

suppressed_message <- " -- Estimates suppressed due to inadequate precision (see <a  target='_blank_'
href = 'https://meps.ahrq.gov/survey_comp/precision_guidelines.shtml'>
FAQs</a> for details).<br>"

rse_message <- " * Relative standard error is greater than 30%.<br>"

notes <- list()

# Statistics -----------------------------------------------------------

notes$care <- notes$ins <- notes$use <- notes$cond <- notes$pmed <- list()

notes$care$pctPOP <- notes$ins$pctPOP <- "
Percentages may not sum to 100 due to rounding.
"

notes$use$totEVT <- notes$use$meanEVT <- notes$use$avgEVT <- "
<h4>Utilization</h4>
Events include all dental visits, prescribed medicine purchases, office-based and outpatient visits, emergency room visits, inpatient stays, and home health events. A <i>home health event</i> is defined as one month during which home health service was received. <i>Other medical equipment and services</i> are not included in utilization estimates because information for these events is collected per interview (e.g. eyeglasses) or per year (e.g. wheelchairs, hearing aids) for each person, rather than on a per-purchase basis.
"

notes$cond$totEVT <- "
<h4>Utilization</h4>
Events include all emergency room visits, home health events, inpatient stays, office-based and outpatient visits, and prescribed medicine purchases associated with a medical condition. A <i>home health event</i> is defined as one month during which home health service was received.
"



notes$use$totEXP <- notes$use$meanEXP <- notes$use$meanEXP0 <- notes$use$medEXP <-
  notes$cond$totEXP <- notes$cond$meanEXP <- "
<h4>Expenditures</h4>
Expenditures include payments for medical events reported during the calendar year. Expenditures in MEPS are defined as the sum of direct payments for care provided during the year, including out-of-pocket payments and payments by private insurance, Medicaid, Medicare, and other sources. Payments for over-the-counter drugs and phone contacts with medical providers are not included in MEPS total expenditure estimates. Indirect payments not related to specific medical events, such as Medicaid Disproportionate Share and Medicare Direct Medical Education subsidies, also are not included. Any charges associated with uncollected liability, bad debt, and charitable care (unless provided by a public clinic or hospital) are not counted as expenditures.
"

notes$use$medEXP <- paste(notes$use$medEXP, "<p>
The median and standard error estimates in this table were produced by the R Programming Language (version 3.3.3). Median and corresponding standard errors produced by different programming languages may differ slightly, due to varying methods for calculating medians and standard errors for survey data.
</p>
")


# Demographics  --------------------------------------------------------

notes$agegrps <- "
<h4>Age groups</h4>
Respondents were asked to report the age of each family member as of the date of each interview for each round of data collection. The age variable used to create these estimates is based on the sample person's age as of the end of the year. If data were not collected during a round because the sample person was out of scope (e.g., deceased or institutionalized), then age at the time of the previous round was used.
"

notes$region <- "
<h4>Region</h4>
The census region variable is based on the location of the household at the end of the year. If missing, the most recent location available is used.
<ul>
  <li><i>Northeast:</i> Maine, New Hampshire, Vermont, Massachusetts, Rhode Island, Connecticut, New York, New Jersey, and Pennsylvania.</li>

  <li><i>Midwest:</i> Ohio, Indiana, Illinois, Michigan, Wisconsin, Minnesota, Iowa, Missouri, North Dakota, South Dakota, Nebraska, and Kansas.</li>

  <li><i>South:</i></i> Delaware, Maryland, District of Columbia, Virginia, West Virginia, North Carolina, South Carolina, Georgia, Florida, Kentucky, Tennessee, Alabama, Mississippi, Arkansas, Louisiana, Oklahoma, and Texas.</li>

  <li><i>West:</i> Montana, Idaho, Wyoming, Colorado, New Mexico, Arizona, Utah, Nevada, Washington, Oregon, California, Alaska, and Hawaii.</li>
</ul>
"

notes$health <- "
<h4>Perceived health status</h4>
<p>The MEPS respondent was asked to rate the health of each person in the family at the time of the interview according to the following categories: excellent, very good, good, fair, and poor. For persons with missing health status in a round, the response for health status at the previous round was used, if available. A small percentage of persons (< 2 percent) had a missing response for <i>perceived health status</i>.</p>
"

notes$mnhlth <- "
<h4>Perceived mental health</h4>
<p>The MEPS respondent was asked to rate the mental health of each person in the family at the time of the interview according to the following categories: excellent, very good, good, fair, and poor. For persons with missing mental health status in a round, the response for mental health status at the previous round was used, if available. A small percentage of persons (< 2 percent) had a missing response for <i>perceived mental health status</i>.</p>
"

notes$married <- "
<h4>Marital status</h4>
Marital status is based on the person's marital status at the end of the year. If missing, the most recent non-missing marital status variable is used. A small percentage of persons (< 2 percent) had a missing value for <i>marital status</i>.
"

notes$education <- "
<h4>Education</h4>
Education for each person is based on the highest education level completed when entering MEPS. A small percentage of persons (< 2 percent) had a missing response for <i>education</i>.
"

notes$employed <- "
<h4>Employment status</h4>
Employment status is based on the person's employment status at the end of the year. If missing, the most recent non-missing employment status variable is used. A small percentage of persons (< 2 percent) had a missing response for <i>employment status</i>.
"

notes$insurance <- notes$ins_lt65 <- notes$ins_ge65 <- "
<h4>Insurance coverage</h4>
<ul>
<li><i>Uninsured:</i>
Individuals who did not have health insurance coverage for the entire calendar year were classified as uninsured. The uninsured were defined as people not covered by Medicaid, Medicare, TRICARE (Armed Forces-related coverage), other public hospital/physician programs, private hospital/physician insurance (including Medigap coverage) or insurance purchased through health insurance Marketplaces. People covered only by non-comprehensive State-specific programs (e.g., Maryland Kidney Disease Program) or private single service plans such as coverage for dental or vision care only, or coverage for accidents or specific diseases, were considered uninsured.
</li>

<li><i>Any private:</i>
Individuals classified as having any private health insurance coverage had private insurance that provided coverage for hospital and physician care (including Medigap coverage and TRICARE) at some point during the year.</li>

<li><i>Public only:</i>
Individuals are considered to have public only health insurance coverage if they were not covered by private insurance or TRICARE and they were covered by Medicare, Medicaid, or other public hospital and physician coverage at some point during the year.</li>

<li><i>65+, No Medicare:</i>
Individuals classified as <i>65+, No Medicare</i> either had private coverage at some point during the year that is not identified as Medigap coverage or were uninsured throughout the year.</li>
</ul>
"

notes$sop <- "
<h4>Source of payment</h4>
<ul>
<li><i>Private:</i> Includes payments made by insurance plans covering hospital and medical care (excluding payments from Medicare, Medicaid, and other public sources). Payments from Medigap plans or TRICARE (Armed-Forces-related coverage) are included.</li>

<li><i>Medicare:</i> A federally financed health insurance plan for the elderly, persons receiving Social Security disability payments, and most persons with end-stage renal disease. Medicare Part A, which provides hospital insurance, is automatically given to those who are eligible for Social Security. Medicare Part B provides supplementary medical insurance that pays for medical expenses and can be purchased for a monthly premium.
</li>

<li><i>Medicaid:</i> A means-tested government program jointly financed by federal and state funds that provides health care to those who are eligible. Program eligibility criteria vary significantly by state, but the program is designed to provide health coverage to families and individuals who are unable to afford necessary medical care.
</li>

<li><i>Other:</i> Includes payments from the Department of Veterans Affairs (excluding TRICARE); other federal sources (Indian Health Service, military treatment facilities, and other care provided by the Federal Government); various state and local sources (community and neighborhood clinics, State and local health departments, and State programs other than Medicaid); payments from Workers' Compensation; and, other unclassified sources (e.g., automobile, homeowner's, or liability insurance, and other miscellaneous or unknown sources). It also includes private insurance payments reported for persons without private health insurance coverage during the year, as defined in MEPS, and Medicaid payments reported for persons who were not enrolled in the Medicaid program at any time during the year.
</li>
</ul>
"

notes$poverty <- "
<h4>Poverty status</h4>
<p>Each sample person was classified according to the total annual income of his or her family. Possible sources of income included annual earnings from wages, salaries, bonuses, tips, and commissions; business and farm gains and losses; unemployment and Worker's Compensation; interest and dividends; alimony, child support, and other private cash transfers; private pensions, individual retirement account (IRA) withdrawals, Social Security, and Department of Veterans Affairs payments; Supplemental Security Income and cash welfare payments from public assistance, Aid to Families with Dependent Children and Aid to Dependent Children; gains or losses from estates, trusts, partnerships, S corporations, rent, and royalties; and a small amount of 'other' income. Poverty status is the ratio of family income to the corresponding federal poverty thresholds, which control for family size and age of the head of family. Categories are defined as follows:</p>
<ul>
  <li><i>Negative or Poor</i>: Household income below the Federal poverty line.</li>

  <li><i>Near poor</i>: Household income over the poverty line through 125 percent of the poverty line.</li>

  <li><i>Low income</i>: Household income 125 percent through 200 percent of the poverty line.</li>

  <li><i>Middle income</i>: over 200 percent to 400 percent of the poverty line.</li>

  <li><i>High income</i>: over 400 percent of the poverty line.</li>
</ul>
"

notes$race <- notes$racesex <- "
<h4>Race/ethnicity</h4>
<p>Classification by race and ethnicity is based on information reported for each family member. Starting in 2002, specifications changed so that individuals could report multiple races. Respondents were asked if the race of the sample person was best described as American Indian, Alaska Native, Asian or Pacific Islander, black, white, or other. Prior to 2002, race categories in the tables for American Indian, Alaska Native, Asian or Pacific Islander, multiple races, white, and other are collapsed into the single category of <i>White and other</i>.</p>

<p>For all years, respondents were asked if the sample person's main national origin or ancestry was Puerto Rican; Cuban; Mexican, Mexicano, Mexican American, or Chicano; other Latin American; or other Spanish. All persons whose main national origin or ancestry was reported in one of these Hispanic groups, regardless of racial background, are classified as Hispanic. Since the Hispanic grouping can include black Hispanic, white Hispanic, and other Hispanic, the race categories of black, white, and other do not include Hispanic people.</p>
"

# Event types are different based on app ('Other medical' not included in conditions app)
ul_event <- "
<h4>Event type</h4>
<ul>
  <li><i>Physician office visits</i> and <i>Non-physician office visits</i> are sub-categories of <i>Office-based events</i>.</li>
  <li><i>Physician hosp. visits</i> and <i>Non-physician hosp. visits</i> are sub-categories of <i>Outpatient events</i>.</li>
  <li>A <i>home health event</i> is defined as one month during which home health service was received.</li>
  <li>For <i>prescription medicines</i>, an event is defined as a purchase or refill.</li>
  %s
</ul>
"

notes$use$event <- sprintf(ul_event,
  '<li><i>Other medical equipment and services</i> are expenses for medical equipment such as eyeglasses, hearing aids, or wheelchairs.</li>')

notes$cond$event <- sprintf(ul_event,"")



# Accessibility and Quality of Care --------------------------------------------------------

notes$usc <- "
<h4>Usual source of care</h4>
For each individual family member, the respondent is asked whether there is a particular doctor's office, clinic, health center, or other place that the individual usually goes to if he/she is sick or needs advice about his/her health.
"

notes$diab_foot <- "
<h4>Foot care</h4>
Starting in 2008, the questionnaire for foot care changed slightly, splitting
<i>No exam in past year</i> into exam <i>More than 1 year ago</i> and
<i>Never had feet checked</i>.
"
notes$diab_eye <- "
<h4>Eye exam</h4>
A small percentage of persons (< 0.1 percent) had a missing or invalid response for eye exam.
"

notes$difficulty <- "
<h4>Difficulty receiving needed care</h4>
Difficulty categories are not mutually exclusive. For instance, a person can have difficulty obtaining both medical and dental care.
"

notes$rsn_ANY <- notes$rsn_MD <- notes$rsn_DN <- notes$rsn_PM <- "
<h4>Reasons for difficulty receiving needed care</h4>
Reasons for difficulty are not mutually exclusive. For instance, a person can have difficulty due to insurance-related issues as well as affordability.
"

# Medical Conditions  --------------------------------------------------------

notes$Condition <- "
<h4>Conditions</h4>
<p>Medical conditions are based on conditions for which treatment was received, where treatment includes emergency room visits, home health care, inpatient stays, office-based visits, outpatient visits, and prescription medicine purchases. <i>Other medical equipment and services</i> and <i>dental visits</i> are not included in these tables since medical conditions are not collected for these event types.</p>

<p>Starting in 2007, new survey questions were introduced into MEPS asking participants about whether they had been told they have certain priority health conditions. This change in the survey methodology may have impacted responses for utilization and expenditures related to the following conditions: hypertension, heart disease, cerebrovascular disease, COPD, asthma, hyperlipidemia, cancer, diabetes mellitus, and osteoarthritis. Care should be taken when analyzing these conditions over a time period that spans the 2007 change in survey methodology.</p>

<p>Details on how condition categories are created can be found in the <a href = https://meps.ahrq.gov/data_stats/conditions.shtml>Condition Categories</a> information table.</p>
"

# Prescribed Drugs ------------------------------------------------------------

notes$RXDRGNAM <- "
Data source for generic drug name is Cerner Multum Inc.
"

notes$TC1name <- '
<p>Data source for therapeutic class is Cerner Multum Inc.</p>

<p>The overwhelming majority of items in the "Not Ascertained" category are medical supplies and devices, such as test strips, lancets, and glucometers.</p>

<p>Users should be cautious when assessing trends in therapeutic classes, because Multum\'s therapeutic classification has changed across the years of the MEPS. The Multum variables on each year of this table reflect the most recent classification available in the year the MEPS Prescribed Medicines files were originally released. Since the release of the 1996 Prescribed Medicines file, the Multum classification has been changed by the addition of new classes, and by drugs switching classes. For example: 1) In 1996-2004, antidiabetic drugs were a subclass of the hormone class, but in subsequent years, the antidiabetic subclass is part of a class of metabolic drugs. 2) In 1996-2004, antihyperlipidemic agents were categorized as a class. In subsequent files, antihyperlipidemic drugs were a subclass in the metabolic class. 3) In 1996-2004, the psychotherapeutic class comprised drugs from four subclasses: antidepressants, antipsychotics, anxiolytics/sedatives/hypnotics, and CNS stimulants. In subsequent files, the psychotherapeutic class comprised only antidepressants and antipsychotics.</p>'

File: mepstrends/hc_ins/json/notes.js
var appKey = 'hc_ins';

var pufNames = {"1996":[{"Year":"1996","PIT":"h01","FYC":"h12","Conditions":"hc006r","PMED":"hc10a","Jobs":"hc007","PRPL":"h24","Longitudinal":"-","CLNK":"hc10if1","RXLK":"hc10if2","Multum":"h68f1","MOS":"-","RX":"hc10a","DV":"hc10bf1","OM":"hc10cf1","IP":"hc10df1","ER":"hc10ef1","OP":"hc10ff1","OB":"hc10gf1","MV":"hc10gf1","HH":"hc10hf1"}],"1997":[{"Year":"1997","PIT":"hc005xf","FYC":"h20","Conditions":"h18","PMED":"h16a","Jobs":"hc19","PRPL":"h47f1","Longitudinal":"h23","CLNK":"h16if1","RXLK":"h16if2","Multum":"h68f2","MOS":"-","RX":"h16a","DV":"hc16bf1","OM":"hc16cf1","IP":"hc16df1","ER":"hc16ef1","OP":"hc16ff1","OB":"hc16gf1","MV":"hc16gf1","HH":"hc16hf1"}],"1998":[{"Year":"1998","PIT":"hc009xf","FYC":"h28","Conditions":"h27","PMED":"h26a","Jobs":"h25","PRPL":"h47f2","Longitudinal":"h35","CLNK":"h26if1","RXLK":"h26if2","Multum":"h68f3","MOS":"-","RX":"h26a","DV":"hc26bf1","OM":"h26cf1","IP":"h26df1","ER":"h26ef1","OP":"h26ff1","OB":"h26gf1","MV":"h26gf1","HH":"h26hf1"}],"1999":[{"Year":"1999","PIT":"hc013xf","FYC":"h38","Conditions":"h37","PMED":"h33a","Jobs":"h32","PRPL":"h47f3","Longitudinal":"h48","CLNK":"h33if1","RXLK":"h33if2","Multum":"h68f4","MOS":"-","RX":"h33a","DV":"h33b","OM":"h33c","IP":"h33d","ER":"h33e","OP":"h33f","OB":"h33g","MV":"h33g","HH":"h33h"}],"2000":[{"Year":"2000","PIT":"h22","FYC":"h50","Conditions":"h52","PMED":"h51a","Jobs":"h40","PRPL":"h47f4","Longitudinal":"h58","CLNK":"h51if1","RXLK":"h51if2","Multum":"h68f5","MOS":"-","RX":"h51a","DV":"h51b","OM":"h51c","IP":"h51d","ER":"h51e","OP":"h51f","OB":"h51g","MV":"h51g","HH":"h51h"}],"2001":[{"Year":"2001","PIT":"h34","FYC":"h60","Conditions":"h61","PMED":"h59a","Jobs":"h56","PRPL":"h57","Longitudinal":"h65","CLNK":"h59if1","RXLK":"h59if2","Multum":"h68f6","MOS":"-","RX":"h59a","DV":"h59b","OM":"h59c","IP":"h59d","ER":"h59e","OP":"h59f","OB":"h59g","MV":"h59g","HH":"h59h"}],"2002":[{"Year":"2002","PIT":"h53","FYC":"h70","Conditions":"h69","PMED":"h67a","Jobs":"h63","PRPL":"h66","Longitudinal":"h71","CLNK":"h67if1","RXLK":"h67if2","Multum":"h68f7","MOS":"-","RX":"h67a","DV":"h67b","OM":"h67c","IP":"h67d","ER":"h67e","OP":"h67f","OB":"h67g","MV":"h67g","HH":"h67h"}],"2003":[{"Year":"2003","PIT":"h64","FYC":"h79","Conditions":"h78","PMED":"h77a","Jobs":"h74","PRPL":"h76","Longitudinal":"h80","CLNK":"h77if1","RXLK":"h77if2","Multum":"h68f8","MOS":"-","RX":"h77a","DV":"h77b","OM":"h77c","IP":"h77d","ER":"h77e","OP":"h77f","OB":"h77g","MV":"h77g","HH":"h77h"}],"2004":[{"Year":"2004","PIT":"h75","FYC":"h89","Conditions":"h87","PMED":"h85a","Jobs":"h83","PRPL":"h88","Longitudinal":"h86","CLNK":"h85if1","RXLK":"h85if2","Multum":"h68f9","MOS":"-","RX":"h85a","DV":"h85b","OM":"h85c","IP":"h85d","ER":"h85e","OP":"h85f","OB":"h85g","MV":"h85g","HH":"h85h"}],"2005":[{"Year":"2005","PIT":"h84","FYC":"h97","Conditions":"h96","PMED":"h94a","Jobs":"h91","PRPL":"h95","Longitudinal":"h98","CLNK":"h94if1","RXLK":"h94if2","Multum":"h68f10","MOS":"-","RX":"h94a","DV":"h94b","OM":"h94c","IP":"h94d","ER":"h94e","OP":"h94f","OB":"h94g","MV":"h94g","HH":"h94h"}],"2006":[{"Year":"2006","PIT":"h93","FYC":"h105","Conditions":"h104","PMED":"h102a","Jobs":"h100","PRPL":"h103","Longitudinal":"h106","CLNK":"h102if1","RXLK":"h102if2","Multum":"h68f11","MOS":"-","RX":"h102a","DV":"h102b","OM":"h102c","IP":"h102d","ER":"h102e","OP":"h102f","OB":"h102g","MV":"h102g","HH":"h102h"}],"2007":[{"Year":"2007","PIT":"h101","FYC":"h113","Conditions":"h112","PMED":"h110a","Jobs":"h108","PRPL":"h111","Longitudinal":"h114","CLNK":"h110if1","RXLK":"h110if2","Multum":"h68f12","MOS":"-","RX":"h110a","DV":"h110b","OM":"h110c","IP":"h110d","ER":"h110e","OP":"h110f","OB":"h110g","MV":"h110g","HH":"h110h"}],"2008":[{"Year":"2008","PIT":"h109","FYC":"h121","Conditions":"h120","PMED":"h118a","Jobs":"h116","PRPL":"h119","Longitudinal":"h122","CLNK":"h118if1","RXLK":"h118if2","Multum":"h68f13","MOS":"-","RX":"h118a","DV":"h118b","OM":"h118c","IP":"h118d","ER":"h118e","OP":"h118f","OB":"h118g","MV":"h118g","HH":"h118h"}],"2009":[{"Year":"2009","PIT":"h117","FYC":"h129","Conditions":"h128","PMED":"h126a","Jobs":"h124","PRPL":"h127","Longitudinal":"h130","CLNK":"h126if1","RXLK":"h126if2","Multum":"h68f14","MOS":"-","RX":"h126a","DV":"h126b","OM":"h126c","IP":"h126d","ER":"h126e","OP":"h126f","OB":"h126g","MV":"h126g","HH":"h126h"}],"2010":[{"Year":"2010","PIT":"h125","FYC":"h138","Conditions":"h137","PMED":"h135a","Jobs":"h133","PRPL":"h136","Longitudinal":"h139","CLNK":"h135if1","RXLK":"h135if2","Multum":"h68f15","MOS":"-","RX":"h135a","DV":"h135b","OM":"h135c","IP":"h135d","ER":"h135e","OP":"h135f","OB":"h135g","MV":"h135g","HH":"h135h"}],"2011":[{"Year":"2011","PIT":"h134","FYC":"h147","Conditions":"h146","PMED":"h144a","Jobs":"h142","PRPL":"h145","Longitudinal":"h148","CLNK":"h144if1","RXLK":"h144if2","Multum":"h68f16","MOS":"-","RX":"h144a","DV":"h144b","OM":"h144c","IP":"h144d","ER":"h144e","OP":"h144f","OB":"h144g","MV":"h144g","HH":"h144h"}],"2012":[{"Year":"2012","PIT":"h143","FYC":"h155","Conditions":"h154","PMED":"h152a","Jobs":"h150","PRPL":"h153","Longitudinal":"h156","CLNK":"h152if1","RXLK":"h152if2","Multum":"h68f17","MOS":"-","RX":"h152a","DV":"h152b","OM":"h152c","IP":"h152d","ER":"h152e","OP":"h152f","OB":"h152g","MV":"h152g","HH":"h152h"}],"2013":[{"Year":"2013","PIT":"h151","FYC":"h163","Conditions":"h162","PMED":"h160a","Jobs":"h158","PRPL":"h161","Longitudinal":"h164","CLNK":"h160if1","RXLK":"h160if2","Multum":"h68f18","MOS":"-","RX":"h160a","DV":"h160b","OM":"h160c","IP":"h160d","ER":"h160e","OP":"h160f","OB":"h160g","MV":"h160g","HH":"h160h"}],"2014":[{"Year":"2014","PIT":"h159","FYC":"h171","Conditions":"h170","PMED":"h168a","Jobs":"h166","PRPL":"h169","Longitudinal":"h172","CLNK":"h168if1","RXLK":"h168if2","Multum":"h68f18","MOS":"-","RX":"h168a","DV":"h168b","OM":"h168c","IP":"h168d","ER":"h168e","OP":"h168f","OB":"h168g","MV":"h168g","HH":"h168h"}],"2015":[{"Year":"2015","PIT":"h167","FYC":"h181","Conditions":"h180","PMED":"h178a","Jobs":"h176","PRPL":"h179","Longitudinal":"h183","CLNK":"h178if1","RXLK":"h178if2","Multum":"h68f18","MOS":"h182","RX":"h178a","DV":"h178b","OM":"h178c","IP":"h178d","ER":"h178e","OP":"h178f","OB":"h178g","MV":"h178g","HH":"h178h"}],"2016":[{"Year":"2016","PIT":"h177","FYC":"h192","Conditions":"h190","PMED":"h188a","Jobs":"h185","PRPL":"h191","Longitudinal":"h193","CLNK":"h188if1","RXLK":"h188if2","Multum":"h68f18","MOS":"h187","RX":"h188a","DV":"h188b","OM":"h188c","IP":"h188d","ER":"h188e","OP":"h188f","OB":"h188g","MV":"h188g","HH":"h188h"}],"2017":[{"Year":"2017","PIT":"","FYC":"","Conditions":"","PMED":"","Jobs":"h195","PRPL":"","Longitudinal":"","CLNK":"","RXLK":"","Multum":"h68f18","MOS":"","RX":"","DV":"h197b","OM":"h197c","IP":"h197d","ER":"h197e","OP":"h197f","OB":"h197g","MV":"h197g","HH":"h197h"}]};

var mepsNotes = {"pctPOP":"\nPercentages may not sum to 100 due to rounding.\n","agegrps":"\n<h4>Age groups<\/h4>\nRespondents were asked to report the age of each family member as of the date of each interview for each round of data collection. The age variable used to create these estimates is based on the sample person's age as of the end of the year. If data were not collected during a round because the sample person was out of scope (e.g., deceased or institutionalized), then age at the time of the previous round was used.\n","region":"\n<h4>Region<\/h4>\nThe census region variable is based on the location of the household at the end of the year. If missing, the most recent location available is used.\n<ul>\n  <li><i>Northeast:<\/i> Maine, New Hampshire, Vermont, Massachusetts, Rhode Island, Connecticut, New York, New Jersey, and Pennsylvania.<\/li>\n\n  <li><i>Midwest:<\/i> Ohio, Indiana, Illinois, Michigan, Wisconsin, Minnesota, Iowa, Missouri, North Dakota, South Dakota, Nebraska, and Kansas.<\/li>\n\n  <li><i>South:<\/i><\/i> Delaware, Maryland, District of Columbia, Virginia, West Virginia, North Carolina, South Carolina, Georgia, Florida, Kentucky, Tennessee, Alabama, Mississippi, Arkansas, Louisiana, Oklahoma, and Texas.<\/li>\n\n  <li><i>West:<\/i> Montana, Idaho, Wyoming, Colorado, New Mexico, Arizona, Utah, Nevada, Washington, Oregon, California, Alaska, and Hawaii.<\/li>\n<\/ul>\n","health":"\n<h4>Perceived health status<\/h4>\n<p>The MEPS respondent was asked to rate the health of each person in the family at the time of the interview according to the following categories: excellent, very good, good, fair, and poor. For persons with missing health status in a round, the response for health status at the previous round was used, if available. A small percentage of persons (< 2 percent) had a missing response for <i>perceived health status<\/i>.<\/p>\n","mnhlth":"\n<h4>Perceived mental health<\/h4>\n<p>The MEPS respondent was asked to rate the mental health of each person in the family at the time of the interview according to the following categories: excellent, very good, good, fair, and poor. For persons with missing mental health status in a round, the response for mental health status at the previous round was used, if available. A small percentage of persons (< 2 percent) had a missing response for <i>perceived mental health status<\/i>.<\/p>\n","married":"\n<h4>Marital status<\/h4>\nMarital status is based on the person's marital status at the end of the year. If missing, the most recent non-missing marital status variable is used. A small percentage of persons (< 2 percent) had a missing value for <i>marital status<\/i>.\n","education":"\n<h4>Education<\/h4>\nEducation for each person is based on the highest education level completed when entering MEPS. A small percentage of persons (< 2 percent) had a missing response for <i>education<\/i>.\n","employed":"\n<h4>Employment status<\/h4>\nEmployment status is based on the person's employment status at the end of the year. If missing, the most recent non-missing employment status variable is used. A small percentage of persons (< 2 percent) had a missing response for <i>employment status<\/i>.\n","ins_ge65":"\n<h4>Insurance coverage<\/h4>\n<ul>\n<li><i>Uninsured:<\/i>\nIndividuals who did not have health insurance coverage for the entire calendar year were classified as uninsured. The uninsured were defined as people not covered by Medicaid, Medicare, TRICARE (Armed Forces-related coverage), other public hospital/physician programs, private hospital/physician insurance (including Medigap coverage) or insurance purchased through health insurance Marketplaces. People covered only by non-comprehensive State-specific programs (e.g., Maryland Kidney Disease Program) or private single service plans such as coverage for dental or vision care only, or coverage for accidents or specific diseases, were considered uninsured.\n<\/li>\n\n<li><i>Any private:<\/i>\nIndividuals classified as having any private health insurance coverage had private insurance that provided coverage for hospital and physician care (including Medigap coverage and TRICARE) at some point during the year.<\/li>\n\n<li><i>Public only:<\/i>\nIndividuals are considered to have public only health insurance coverage if they were not covered by private insurance or TRICARE and they were covered by Medicare, Medicaid, or other public hospital and physician coverage at some point during the year.<\/li>\n\n<li><i>65+, No Medicare:<\/i>\nIndividuals classified as <i>65+, No Medicare<\/i> either had private coverage at some point during the year that is not identified as Medigap coverage or were uninsured throughout the year.<\/li>\n<\/ul>\n","ins_lt65":"\n<h4>Insurance coverage<\/h4>\n<ul>\n<li><i>Uninsured:<\/i>\nIndividuals who did not have health insurance coverage for the entire calendar year were classified as uninsured. The uninsured were defined as people not covered by Medicaid, Medicare, TRICARE (Armed Forces-related coverage), other public hospital/physician programs, private hospital/physician insurance (including Medigap coverage) or insurance purchased through health insurance Marketplaces. People covered only by non-comprehensive State-specific programs (e.g., Maryland Kidney Disease Program) or private single service plans such as coverage for dental or vision care only, or coverage for accidents or specific diseases, were considered uninsured.\n<\/li>\n\n<li><i>Any private:<\/i>\nIndividuals classified as having any private health insurance coverage had private insurance that provided coverage for hospital and physician care (including Medigap coverage and TRICARE) at some point during the year.<\/li>\n\n<li><i>Public only:<\/i>\nIndividuals are considered to have public only health insurance coverage if they were not covered by private insurance or TRICARE and they were covered by Medicare, Medicaid, or other public hospital and physician coverage at some point during the year.<\/li>\n\n<li><i>65+, No Medicare:<\/i>\nIndividuals classified as <i>65+, No Medicare<\/i> either had private coverage at some point during the year that is not identified as Medigap coverage or were uninsured throughout the year.<\/li>\n<\/ul>\n","insurance":"\n<h4>Insurance coverage<\/h4>\n<ul>\n<li><i>Uninsured:<\/i>\nIndividuals who did not have health insurance coverage for the entire calendar year were classified as uninsured. The uninsured were defined as people not covered by Medicaid, Medicare, TRICARE (Armed Forces-related coverage), other public hospital/physician programs, private hospital/physician insurance (including Medigap coverage) or insurance purchased through health insurance Marketplaces. People covered only by non-comprehensive State-specific programs (e.g., Maryland Kidney Disease Program) or private single service plans such as coverage for dental or vision care only, or coverage for accidents or specific diseases, were considered uninsured.\n<\/li>\n\n<li><i>Any private:<\/i>\nIndividuals classified as having any private health insurance coverage had private insurance that provided coverage for hospital and physician care (including Medigap coverage and TRICARE) at some point during the year.<\/li>\n\n<li><i>Public only:<\/i>\nIndividuals are considered to have public only health insurance coverage if they were not covered by private insurance or TRICARE and they were covered by Medicare, Medicaid, or other public hospital and physician coverage at some point during the year.<\/li>\n\n<li><i>65+, No Medicare:<\/i>\nIndividuals classified as <i>65+, No Medicare<\/i> either had private coverage at some point during the year that is not identified as Medigap coverage or were uninsured throughout the year.<\/li>\n<\/ul>\n","poverty":"\n<h4>Poverty status<\/h4>\n<p>Each sample person was classified according to the total annual income of his or her family. Possible sources of income included annual earnings from wages, salaries, bonuses, tips, and commissions; business and farm gains and losses; unemployment and Worker's Compensation; interest and dividends; alimony, child support, and other private cash transfers; private pensions, individual retirement account (IRA) withdrawals, Social Security, and Department of Veterans Affairs payments; Supplemental Security Income and cash welfare payments from public assistance, Aid to Families with Dependent Children and Aid to Dependent Children; gains or losses from estates, trusts, partnerships, S corporations, rent, and royalties; and a small amount of 'other' income. Poverty status is the ratio of family income to the corresponding federal poverty thresholds, which control for family size and age of the head of family. Categories are defined as follows:<\/p>\n<ul>\n  <li><i>Negative or Poor<\/i>: Household income below the Federal poverty line.<\/li>\n\n  <li><i>Near poor<\/i>: Household income over the poverty line through 125 percent of the poverty line.<\/li>\n\n  <li><i>Low income<\/i>: Household income 125 percent through 200 percent of the poverty line.<\/li>\n\n  <li><i>Middle income<\/i>: over 200 percent to 400 percent of the poverty line.<\/li>\n\n  <li><i>High income<\/i>: over 400 percent of the poverty line.<\/li>\n<\/ul>\n","racesex":"\n<h4>Race/ethnicity<\/h4>\n<p>Classification by race and ethnicity is based on information reported for each family member. Starting in 2002, specifications changed so that individuals could report multiple races. Respondents were asked if the race of the sample person was best described as American Indian, Alaska Native, Asian or Pacific Islander, black, white, or other. Prior to 2002, race categories in the tables for American Indian, Alaska Native, Asian or Pacific Islander, multiple races, white, and other are collapsed into the single category of <i>White and other<\/i>.<\/p>\n\n<p>For all years, respondents were asked if the sample person's main national origin or ancestry was Puerto Rican; Cuban; Mexican, Mexicano, Mexican American, or Chicano; other Latin American; or other Spanish. All persons whose main national origin or ancestry was reported in one of these Hispanic groups, regardless of racial background, are classified as Hispanic. Since the Hispanic grouping can include black Hispanic, white Hispanic, and other Hispanic, the race categories of black, white, and other do not include Hispanic people.<\/p>\n","race":"\n<h4>Race/ethnicity<\/h4>\n<p>Classification by race and ethnicity is based on information reported for each family member. Starting in 2002, specifications changed so that individuals could report multiple races. Respondents were asked if the race of the sample person was best described as American Indian, Alaska Native, Asian or Pacific Islander, black, white, or other. Prior to 2002, race categories in the tables for American Indian, Alaska Native, Asian or Pacific Islander, multiple races, white, and other are collapsed into the single category of <i>White and other<\/i>.<\/p>\n\n<p>For all years, respondents were asked if the sample person's main national origin or ancestry was Puerto Rican; Cuban; Mexican, Mexicano, Mexican American, or Chicano; other Latin American; or other Spanish. All persons whose main national origin or ancestry was reported in one of these Hispanic groups, regardless of racial background, are classified as Hispanic. Since the Hispanic grouping can include black Hispanic, white Hispanic, and other Hispanic, the race categories of black, white, and other do not include Hispanic people.<\/p>\n"};
